docs/topics/i18n/translation.txt

```diff
@@ -1027,6 +1027,31 @@ directories listed in :setting:`LOCALE_PATHS` have the highest precedence with
 the ones appearing first having higher precedence than the ones appearing
 later.
 
+.. note::
```

Comment:
The note usage is a bit odd. I'd suggest to have it as a normal section that mirrors how javascript_catalog is setup.